qa/standalone/test_ceph_daemon.sh

```diff
@@ -11,7 +11,12 @@ IMAGE_MASTER=${IMAGE_MASTER:-'ceph/daemon-base:latest-master-devel'}
 IMAGE_NAUTILUS=${IMAGE_NAUTILUS:-'ceph/daemon-base:latest-nautilus'}
 IMAGE_MIMIC=${IMAGE_MIMIC:-'ceph/daemon-base:latest-mimic'}
 
-TEST_TARS=${SCRIPT_DIR}/test_ceph_daemon/*.tgz
+CORPUS_GIT_SUBMOD="ceph-daemon-adoption-corpus"
+TMPDIR=$(mktemp -d)
+trap "rm -rf $TMPDIR" EXIT
+git clone https://github.com/ceph/$CORPUS_GIT_SUBMOD $TMPDIR
+CORPUS_DIR=${TMPDIR}/archive
+TEST_TARS=$(find ${CORPUS_DIR} -type f -iname *.tgz)
```

Comment:
```
2019-12-08T20:46:46.433 INFO:tasks.workunit.client.0.smithi186.stderr:++ find /home/ubuntu/cephtest/clone.client.0/qa/standalone/../../ceph-daemon-adoption-corpus/archive -type f -iname '*.tgz'
2019-12-08T20:46:46.434 INFO:tasks.workunit.client.0.smithi186.stderr:find: ‘/home/ubuntu/cephtest/clone.client.0/qa/standalone/../../ceph-daemon-adoption-corpus/archive’: No such file or directory
2019-12-08T20:46:46.435 INFO:tasks.workunit.client.0.smithi186.stderr:+ TEST_TARS=
2019-12-08T20:46:46.435 DEBUG:teuthology.orchestra.run:got remote process result: 1
```

see http://pulpito.ceph.com/ideepika-2019-12-08_20:21:40-rados-wip-deepika-testing-5-12-2019-distro-basic-smithi/4582083/

Reply:
This needs a rebase:
```
2019-12-08T20:46:14.488 INFO:teuthology.orchestra.run.smithi186:> rm -rf /home/ubuntu/cephtest/clone.client.0 && git clone https://github.com/ceph/ceph-ci.git /home/ubuntu/cephtest/clone.client.0 && cd /home/ubuntu/cephtest/clone.client.0 && git checkout 41d84e533c78c67be1676ebf316394233fededce
```

To include this commit: https://github.com/ceph/ceph/pull/31772/commits/acd80573e5e013d0066e95d1d015db4199b86fc5
```
2019-12-08T20:46:46.433 INFO:tasks.workunit.client.0.smithi186.stderr:+ CORPUS_GIT_SUBMOD=ceph-daemon-adoption-corpus
2019-12-08T20:46:46.433 INFO:tasks.workunit.client.0.smithi186.stderr:+ CORPUS_DIR=/home/ubuntu/cephtest/clone.client.0/qa/standalone/../../ceph-daemon-adoption-corpus/archive
2019-12-08T20:46:46.433 INFO:tasks.workunit.client.0.smithi186.stderr:++ find /home/ubuntu/cephtest/clone.client.0/qa/standalone/../../ceph-daemon-adoption-corpus/archive -type f -iname '*.tgz'
2019-12-08T20:46:46.434 INFO:tasks.workunit.client.0.smithi186.stderr:find: â€˜/home/ubuntu/cephtest/clone.client.0/qa/standalone/../../ceph-daemon-adoption-corpus/archiveâ€™: No such file or directory
2019-12-08T20:46:46.435 INFO:tasks.workunit.client.0.smithi186.stderr:+ TEST_TARS=
```

Reply:
Please also include PR #32137 during the next test run too...